app/controllers/api/physical_servers_controller.rb

```diff
@@ -64,14 +75,23 @@ def change_resource_state(state, type, id)
           server = resource_search(id, type, klass)
           desc = "Requested server state #{state} for #{server_ident(server)}"
           api_log_info(desc)
-          task_id = queue_object_action(server, desc, :method_name => state, :role => :ems_operations)
+          task_id = queue_object_action(server, desc, :method_name => state, :role => :ems_operations, :args => data)
           action_result(true, desc, :task_id => task_id)
-        rescue => err
+        rescue StandardError => err
           action_result(false, err.to_s)
         end
       end
     end
 
+    # Process the check of the customization script by id
+    #
+    # @raise [NotFoundError] if id is invalid
+    def ensure_customization_script_exists(id)
+      ensure_resource_exists(:customization_scripts, id)
```

Comment:
Instead of a wrapper method around `ensure_resource_exists` for this particular case, I think it may make sense to add a message such as `"#{type} not found"` in `ensure_resource_exists` when the `NotFoundError` is raised. Thoughts?

Reply:
Yes. It makes sense